.env

```diff
@@ -24,3 +24,7 @@ HTTP_GW_IMAGE=nspccdev/neofs-http-gate
 # S3 Gate
 S3_GW_VERSION=0.14.2
 S3_GW_IMAGE=nspccdev/neofs-s3-gate
+
+# NeoFS LOCODE database
+#LOCODE_DB_URL=<url>
+#LOCODE_DB_PATH=/path/to/locode_db
```

Comment:
In the end, when db will be available online, just keep only URL the same way it is done for chain